R/industry_subsectors_specific.R

```diff
@@ -49,34 +49,38 @@
 #' @export
 #' @rdname industry_subsector_specific
 readindustry_subsectors_specific <- function(subtype = NULL) {
```

Comment:
could you rename the function and the file to readIndustrySubsectorsSpecific

Reply:
Why?

Reply:
> The dependency graph only considers madrat-style functions, e.g. functions not starting with `download`, `read`, `correct`, `convert`, `calc`, or `tool` will not be considered.

[—`madrat::madrat-caching`](https://github.com/pik-piam/madrat/blob/88c4cdf2d91a145712899ed0d3dd4e5b867a7813/vignettes/madrat-caching.Rmd#L50)

All function names follow this convention.